styleguide/components/31-molecules/menu/_menu.scss

```diff
@@ -157,7 +161,9 @@ nav.menu {
         &.active,
         &:hover,
         &:focus {
+          outline: 0;
```

Comment:
:scream: Is this really necessary?

Reply:
👍

Reply:
Rob didn't liked it 🙈

Reply:
That's not a good enough reason... Outline has a specific purpose. I think we should consider this during the design phase... Just because someone doesn't like the way it looks now is no reason te remove it. @delrueba ?

Reply:
Yes... this stuff gives me nightmares. 
However, it seems this specific outline behaves a bit funky since the target zone has been enlarged.

So, if the focusstyle is clearly different: OK for me.